spaces/domains.json

```
@@ -53,13 +53,16 @@
   "vote.frontier.xyz": "frontier",
   "vote.oceanprotocol.com": "oceandao",
   "vote.sav3.org": "sav3",
+  "vote.bt.finance": "btfinance.eth",
+  "vote.zlot.finance": "zlot"
   "vote.zlot.finance": "zlot",
```

When you added your domain, you duplicated 2 lines in domain.json
```suggestion
  "vote.bt.finance": "btfinance.eth",
  "vote.zlot.finance": "zlot",
```